accepted/targeting-packs-and-runtime-packs.md

```diff
@@ -0,0 +1,286 @@
+* Delivered via windows installers, chained in to Visual Studio
+
+**2. Framework NuGet packages**
+* Used by "modern" frameworks (.NET Core, .NET Standard, UWP)
+* Provide both reference assemblies and implementation as standard nuget assets
+* Downloaded via nuget restore if not already in fallback folder/cache
+
+And each have had strengths and wekaness:
+
+**Traditional targeting pack strengths**
+* Fast build performance
+* Obvious separation of platform vs. ecosystem
+* Globally resolveable without project context (depended upon by certain VS features)
+
+**Traditional targeting pack weaknesses**
+* Windows-only acquisition
+* Admin-only acquisition
+* Manual installation
+* Infamous error experience when not installed
+* No self-contained deployment strategy
+
+**Framework NuGet package stregnths**
+* Cross-platform/cross-IDE acquisition
+* Non-admin acquisition
+* Seamless acquisition: just clone, restore, and build
+* Allow for (cross-targeted) self-contained deployment
+
+**Framework Nuget package weaknesses**
+* Slow build (restore) performance
+* Extremely complex NuGet graph that leaks to customers
+* Large size on disk
+* Difficult to understand distinction between platform and ecosystem
+* Difficult to predict when network access is required
+* Requires project context for resolution
+
+
+# Summary of New Approach
+
+We introduce a third, hybrid approach to framework references that aims to capture the strengths of the earlier approaches while mitigating weaknesses. 
+
+In this new approach, framework reference assemblies will be commonly resolved from a global location alongside `dotnet`. This is similar in spirit to the classic targeting packs in `C:\Program Files\Reference Assemblies`, but cross-platform and compatible with non-admin SDK deployment.
+
+On top of that, we layer a fallback to acquiring the same assets via NuGet. This will be used so that you can easily build projects targeting frameworks that are older than the SDKs you have installed. 
+
+Finally, we add a new `FrameworkReference` concept to the build, to represent the use of a group of assemblies that version with the target framework.
+
+
+# Targeting, Runtime, and AppHost Packs
+
+We use the term "pack" for a collection of files used by the build. A pack can either be deployed globally alongside `dotnet` or wrapped in a NuGet package.
+
+There are three categories of packs:
+
+1. Targeting packs - reference assemblies, documentation, and other design-time assets
+2. Runtime packs - runtime assets for self-contained publish
+3. AppHost packs - app executable template to generate native app executable
+
+
+# Globally Installed Packs 
+
+Targeting packs (and AppHost packs) will be bundled with the SDK, and available to be installed globally via native installers. This bundling is directly analagous to how the shared frameworks are bundled with the SDK or installable individually.
```

Comment:
How does the targeting pack become part of a [source-build](https://github.com/dotnet/source-build) .NET Core?

Reply:
cc @crummel @dagood @dseefeld

Reply:
I would say It should work just like the shared frameworks work, however that is.  :) There's a collection of files that will get produced that go in the packs/ folder just like a set of files that go in the shared/ folder. The source code for producing both sets is available.